.github/workflows/main.yml

```diff
@@ -1,10 +1,6 @@
 name: CI
```

Comment:
If this only runs on a pull request then maybe we should update the name and the filename of this workflow?

Although, should we keep running this on pushes to `develop` to ensure the working branch is always healthy?

Reply:
Maybe instead of just `pull-request` we do `push` to branches `develop` and `main`? I think we should have CI run on any PR, to make sure we're not breaking what we push it into... But if you think we should only run it on `develop` and `main` then I am happy for that instead of every PR.

I'll change the filename to `ci.yml`?

Reply:
Maybe we could just do it on every push to any branch? That way pull requests would be open against commits in that branch that had already run it rather than running it on a pull request as well?

I'm thinking it might be best to rename this to `ci.yml` and just have:

```yml
on:
  push:
```

Keen to see what other devs think...

Reply:
Gotcha, yep, that sounds best to me!😄